insights/specs/default.py

```diff
@@ -557,6 +557,7 @@ def semid(broker):
     nfnetlink_queue = simple_file("/proc/net/netfilter/nfnetlink_queue")
     nfs_exports = simple_file("/etc/exports")
     nfs_exports_d = glob_file("/etc/exports.d/*.exports")
+    nft_ruleset = simple_command("/usr/sbin/nft list ruleset")
```

Comment:
You need to include an archive spec for this command spec so that the information can be read from an insights archive.